src/screens/FeedScreen/RenderList.js

```diff
@@ -244,7 +269,8 @@ RenderListFeed.propTypes = {
   hideThreeDot: PropTypes.bool,
   showAnonymousOption: PropTypes.bool,
   onDeletePost: PropTypes.func,
-  isSelf: PropTypes.bool
+  isSelf: PropTypes.bool,
+  onPreviewCommentPress: PropTypes.func
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onPreviewCommentPress` prop type is added but it's not used anywhere in the provided code. If it's not used in other parts of the file, consider removing this prop type to keep the code clean and maintainable.
```diff
- onPreviewCommentPress: PropTypes.func
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->